CesiumGltfReader/CMakeLists.txt

```diff
@@ -37,16 +37,18 @@ target_sources(
     PRIVATE
         ${CESIUM_GLTF_READER_SOURCES}
         ${CESIUM_GLTF_READER_HEADERS}
-    PUBLIC
         ${CESIUM_GLTF_READER_PUBLIC_HEADERS}
 )
 
 cesium_target_include_directories(
     TARGET
         CesiumGltfReader
     PUBLIC
-        ${CMAKE_CURRENT_LIST_DIR}/include
-        ${CMAKE_CURRENT_LIST_DIR}/generated/include
+        # Necessary for `draco/draco_features.h`
+        $<BUILD_INTERFACE:${CESIUM_NATIVE_DRACO_INCLUDE_DIR}>
```

Comment:
Another occurrence of this. It has a comment this time, but it doesn't clear things up for me. We don't appear to be including `draco/draco_features.h` anywhere?